deployment-scanner/src/main/java/org/jboss/as/server/deployment/scanner/FileSystemDeploymentService.java

```diff
@@ -781,16 +859,14 @@ private void scanDirectory(final File directory, final String relativePath, fina
                                 if (isZipComplete(child)) {
                                     final String path = relativeTo == null ? child.getAbsolutePath() : relativePath + fileName;
                                     final boolean archive = child.isFile();
-                                    if(firstScan){
+                                    if (firstScan) {
                                         scanContext.firstScanDeployments.add(fileName);
                                     }
                                     addContentAddingTask(path, archive, fileName, child, timestamp, scanContext);
-                                } else {
-                                    //we need to make sure that the file was not deleted while
-                                    //the scanner was running
-                                    if (child.exists()) {
-                                        scanContext.incompleteFiles.put(child, new IncompleteDeploymentStatus(child, timestamp));
-                                    }
+                                } else //we need to make sure that the file was not deleted while
+                                //the scanner was running
+                                if (child.exists()) {
+                                    scanContext.incompleteFiles.put(child, new IncompleteDeploymentStatus(child, timestamp));
```

Comment:
This is ugly. :)

Please configure your IDE to not change whitespace or formatting on unmodified lines.